app/src/main/kotlin/org/cru/godtools/ui/dashboard/DashboardActivity.kt

```diff
@@ -77,10 +118,16 @@ class DashboardActivity : BaseActivity() {
             intent.action == Intent.ACTION_VIEW -> {
                 val data = intent.data
                 when {
-                    data?.isDashboardCustomUriSchemeDeepLink() == true ->
-                        viewModel.updateCurrentPage(findPageByUriPathSegment(data.pathSegments.getOrNull(1)))
-                    data?.isDashboardGodToolsDeepLink() == true ->
-                        viewModel.updateCurrentPage(findPageByUriPathSegment(data.pathSegments.getOrNull(2)))
+                    data?.isDashboardCustomUriSchemeDeepLink() == true -> viewModel.updateCurrentPage(
+                        findPageByUriPathSegment(data.pathSegments.getOrNull(1))
+                    )
+
+                    data?.isDashboardGodToolsDeepLink() == true -> viewModel.updateCurrentPage(
+                        findPageByUriPathSegment(
+                            data.pathSegments.getOrNull(2)
+                        )
+                    )
```

Comment:
more auto-formatter shenanigans, I don't think this logic changed at all and was perfectly fine with ktlint rules defined